experiments/amg2023/openmp/ramble.yaml

```diff
@@ -45,7 +45,8 @@ ramble:
   spack:
     packages:
       hypre-omp:
-        spack_spec: hypre@2.31.0 +mpi+openmp+mixedint~fortran{modifier_spack_variant}
+        # spack_spec: hypre@2.31.0 +mpi+openmp+mixedint~fortran{modifier_spack_variant}
```

Comment:
I generally prefer removing code/params that are commented out, but I can see the point in leaving them if it's likely you will revert to the original value at some point in the future. In that case, you may want to add a short comment to that effect This comment applies to several files in this comment, but I'll just note it here.